libs/domain/cart/src/services/adapter/cart.adapter.ts

```diff
@@ -13,6 +14,7 @@ export interface CartAdapter {
   addEntry: (data: AddCartEntryQualifier) => Observable<Cart>;
   deleteEntry: (data: CartEntryQualifier) => Observable<unknown>;
   updateEntry: (data: UpdateCartEntryQualifier) => Observable<Cart>;
+  updateCartPriceMode: (data: UpdateCartPriceModeQualifier) => Observable<Cart>;
```

Comment:
Let's create it more generic like `update` cart
```suggestion
  update: (data: UpdateCartQualifier) => Observable<Cart>;
```